policyengine_us/parameters/gov/states/mt/tax/income/subtractions/disability_income/disability_income_cap.yaml

```diff
@@ -0,0 +1,11 @@
+description: Montana caps the disability income at this amount.
+values: 
+  2021-01-01: 5_200
+
+metadata:
+  label: Montana disability income exclusion cap
+  period: year
+  unit: currency-USD
+  reference:
+  - title: 2022 Montana form 2 individual income tax instructions, page 31, line 32
```

Comment:
need to see if there is a legal reference - if there is no add a comment stating that it is not mentioned 

File an issue called "Reconcile the Montana disability income subtraction" - explain that legal code does not mention the cap + leave references